src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -25,16 +25,25 @@ function logCapturedError(capturedError : CapturedError) : void {
       willRetry,
     } = capturedError;
 
-    const errorSummary = error.message
-      ? `Error: ${error.message}\n\n`
+    const {
+      message,
+      name,
+      stack,
+    } = error;
+
+    const errorSummary = message
+      ? `${name}: ${message}\n\n`
       : '';
 
     const componentNameMessage = componentName
       ? `React caught an error thrown by ${componentName}.`
       : 'React caught an error thrown by one of your components.';
 
-    const callStack = error.stack;
-    const trimmedCallStack = callStack.substring(callStack.indexOf('\n'));
+    // Error stack varies by browser; normalize it for our logging.
+    let formattedCallStack = stack.indexOf(message) >= 0
```

Comment:
I think it would be safer to split the first line first. Then check *the first line* for matching the message.
Otherwise this code behave in a weird way if the message was not on the first line.

Now that I think of it, messages could also be multiline. So maybe it's best to check for a very specific pattern that Chrome does, and not touch it in any other case.